sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -590,7 +590,8 @@ export class ApplicationControllerService {
       applicationStatus: applicationDetail.applicationStatus,
       applicationStatusUpdatedOn: applicationDetail.applicationStatusUpdatedOn,
       applicationNumber: applicationDetail.applicationNumber,
-      applicationOfferingIntensity: offering?.offeringIntensity,
+      applicationOfferingIntensity:
```

Comment:
There are logics, like below, relieing on the fact that `applicationOfferingIntensity` will be present only when the offering dates are present, like the below.

https://github.com/bcgov/SIMS/blob/fcfdd841c9bfbb165b539564eef921eea2b19349/sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue#L32-L36